quickstart.sh

```diff
@@ -86,12 +86,114 @@ docker compose up -d --build ${services}
 echo -e "${MAJOR}Waiting for OpenSearch to start up and be online.${RESET}"
 ./opensearch/wait-for-os.sh # Wait for OpenSearch to be online
 
-echo -e "${MAJOR}Creating ecommerce-keyword index, defining its mapping & settings\n${RESET}"
-curl -s -X PUT "http://localhost:9200/ecommerce-keyword/" -H 'Content-Type: application/json' --data-binary @./opensearch/schema.json
-echo -e "\n"
+echo -e "${MAJOR}Configuring the ML Commons plugin.${RESET}"
+curl -s -X PUT "http://localhost:9200/_cluster/settings" -H 'Content-Type: application/json' --data-binary '{
+  "persistent": {
+        "plugins": {
+            "ml_commons": {
+                "only_run_on_ml_node": "false",
+                "model_access_control_enabled": "true",
+                "native_memory_threshold": "99"
+            }
+        }
+    }
+}'
+
+echo -e "${MAJOR}Registering a model group.${RESET}"
+response=$(curl -s -X POST "http://localhost:9200/_plugins/_ml/model_groups/_register" \
+  -H 'Content-Type: application/json' \
+  --data-binary '{
+    "name": "neural_search_model_group",
+    "description": "A model group for neural search models"
+  }')
+
+# Extract the model_group_id from the JSON response
+model_group_id=$(echo "$response" | jq -r '.model_group_id')
+
+# Use the extracted model_group_id
+echo "Created Model Group with id: $model_group_id"
+
+echo -e "${MAJOR}Registering a model in the model group.${RESET}"
+response=$(curl -s -X POST "http://localhost:9200/_plugins/_ml/models/_register" \
+  -H 'Content-Type: application/json' \
+  --data-binary "{
```

Comment:
i don't think we need all the `\` characters..   We don't have that in other examples...  where the `--data-binary` uses a `'`

Reply:
see https://github.com/o19s/chorus-opensearch-edition/pull/127#issuecomment-2516471695
Pretty sure we need the backslashes. Otherwise the variables are not resolved and treated as string literals.